dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/dxf2/deprecated/tracker/EventImportTest.java

```diff
@@ -279,12 +279,12 @@ void shouldUpdateEventDataValues_whenAddingDataValuesToEvent() throws IOExceptio
     // add a new data value and update an existing one
 
     DataValue dataValueA = new DataValue();
-    dataValueA.setValue("10'000");
+    dataValueA.setValue("10'''000'''");
```

Comment:
Do we assert anywhere that our replace logic does not alter what the user wanted to store?

Reply:
yes there are assertions later referring to the getValue